README.md

```diff
@@ -1,24 +1,71 @@
-# README
+# I-spend
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- ROR
+- Postgresql
 
-* System dependencies
+## Getting Started
 
-* Configuration
+0.0 make sure to install ruby on your machine.
 
-* Database creation
+1. Open [dev branch](https://github.com/Tadesse-Alemayehu/I-spend)
+2. Open VSCode or terminal and navigate to the folder where you would like to clone the project.
+3. Type "pwd" to see your working directory.
+4. Clone repository.
 
-* Database initialization
+```bash
+git clone git@github.com:marilirulita/I-spend
+```
 
-* How to run the test suite
+5. Install gems by running.
 
-* Services (job queues, cache servers, search engines, etc.)
+```bash
+bundle install
+```
 
-* Deployment instructions
+6. Start the server by running.
 
-* ...
+```bash
+rails s
+```
+
+7. Enjoy 😎
+
+### Prerequisites
+
+> You must have Ruby installed on your computer.
+>
+> You must have rails installed on your computer.
+
+## Go Live 😎
+
+[I-spend](https://quiet-beach-52239.herokuapp.com/)
+
+## Authors
+
+👤 **Tadesse Alemayehu**
+
+- GitHub: [@Tadess-Alemayehu](https://github.com/Tadesse-Alemayehu)
+- Twitter: [@TadesseWebDev](https://twitter.com/TadesseWebDev)
+- LinkedIn: [Tadesse Alemayehu](https://www.linkedin.com/in/tadesse-alemayehu-60141a221/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to all our code reviewers and partners at Microverse.
+
```

Comment:
- [ ] The Acknowledgment section should give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Reply:
Thanks, @medaminedev66. Ill make sure to do that.